e2e-polybft/cardanofw/test-cardano-chain.go

```diff
@@ -184,14 +184,17 @@ func (ec *TestCardanoChain) CreateAddresses(
 	return nil
 }
 
-func (ec *TestCardanoChain) FundWallets(ctx context.Context) error {
+func (ec *TestCardanoChain) FundWallets(ctx context.Context, fundAmount *big.Int) error {
 	genesisWallet, err := GetGenesisWalletFromCluster(ec.cluster.Config.TmpDir, 1)
 	if err != nil {
 		return err
 	}
 
 	privateKey := hex.EncodeToString(genesisWallet.GetSigningKey())
-	fundAmount := new(big.Int).SetUint64(ec.config.FundAmount)
+
+	if fundAmount == nil || fundAmount.Cmp(big.NewInt(0)) == 0 {
```

Comment:
i would remove zero check

Reply:
also...probably would not fund relayer too in this case. @babinskizoltan ?

Reply:
hot wallet is funded either way, this just changes the amount. So why wouldn't we fund relayer? It would be different if there was a check here, if ec.Config.FundAmount was zero, and in that case I wouldnt fund either addresses. Also, that would be better imo, since as things are now, we must specify minUtxo for the initial fund, and then we must send more than that through bridging requests, so that they would be rejected. Also, as things are now, we have 2 hotWalletIncrement claims per chain, which is fine, but not necessary